integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -25,7 +25,7 @@ class MoodleContentMetadataExporter(ContentMetadataExporter):
         'categoryid': 'categoryid',
     }
 
-    LONG_STRING_LIMIT = 2000
+    LONG_STRING_LIMIT = 500
```

Comment:
so for Moodlecloud, the 2000 length limit + 5 courses worked successfully, but I feel like some combination of lower description limit and item slicing could be better? Idk. Maybe having the full description is _that_ important.